tiatoolbox/wsicore/wsireader.py

```diff
@@ -951,7 +969,7 @@ def _read_rect_at_resolution(
         location: NumPair,
         size: NumPair,
         resolution: Resolution = 0,
-        units: str = "level",
+        units: Units = "level",
```

Comment:
Same as above. These kind of edits can be suggested in a separate PR.

Reply:
Moved to https://github.com/TissueImageAnalytics/tiatoolbox/pull/598